redbot/cogs/mod/names.py

```diff
@@ -245,15 +245,25 @@ async def userinfo(self, ctx, *, user: discord.Member = None):
         data.add_field(name=_("Joined Discord on"), value=created_on)
         data.add_field(name=_("Joined this server on"), value=joined_on)
         if role_str is not None:
-            data.add_field(name=_("Roles"), value=role_str, inline=False)
+            data.add_field(
+                name=_("Roles") if len(roles) > 2 else _("Role"), value=role_str, inline=False
+            )  # len(roles) > 2 because @everyone is a "role" which is just not displayed.
```

Comment:
Sorry, I somehow missed this. We already strip the `@everyone` role above (see line 173), so this results in incorrect results.
```suggestion
                name=_("Roles") if len(roles) > 1 else _("Role"), value=role_str, inline=False
            )
```